changelogs/fragments/fix-default-ansible-galaxy-role-import-name.yml

```diff
@@ -0,0 +1,8 @@
+bugfixes:
+  - >-
+    ``ansible-galaxy role import`` - fix using the ``role_name`` in a standalone role's
+    ``galaxy_info`` metadata by removing the CLI magic remove the ``ansible-role-`` prefix.
```

Comment:
This is very difficult to read and understand. Maybe something more like
```suggestion
    ``galaxy_info`` metadata by disabling automatic removal of the ``ansible-role-`` prefix.
```

Reply:
Thanks, I was debating fixing that :smiley: